### ISSUE TITLE
blt vm adds drupal-vm to require instead of require-dev

### ISSUE THREAD

**Issue by CashWilliams** [acquia/blt#2127]
My system information:
* BLT version: 8.9.8

When I run this command:
```
blt vm
```

Composer.json contains:
```
   "require": {
        "acquia/blt": "^8.3",
        "geerlingguy/drupal-vm": "~4.3"
    },
```

And I expected composer.json to contain:
```
   "require": {
        "acquia/blt": "^8.3"
   },
   "require-dev": {
        "geerlingguy/drupal-vm": "~4.3"
    },
```

**Comment by grasmash** [comment]
I'm pretty sure this is fixed in 9.1.x.

**Comment by grasmash** [comment]
This has been fixed and will be released in the next version of 9.1.x.
